plugins/repository-azure/src/main/java/org/elasticsearch/cloud/azure/blobstore/AzureBlobContainer.java

```diff
@@ -72,14 +72,14 @@ public InputStream readBlob(String blobName) throws IOException {
         logger.trace("readBlob({})", blobName);
 
         if (!blobExists(blobName)) {
```

Comment:
@gfyoung what do you mean by lenient? What does it do if the file doesn't exist and you are trying to read the file?